database/src/adapters/receipts.rs

```diff
@@ -486,14 +486,43 @@ async fn save_receipts(
 
 async fn store_receipt_actions(
```

Comment:
```suggestion
async fn store_action_receipts(
```
I wonder if we can do a similar renaming here, like you have with data receipts. This conflicts with the new method you already created, so maybe we can just merge that logic in here?

Not sure which is better but I like the idea of avoiding use of `receipt_action` and `receipt_data` for naming action/data receipts. I easily confuse them with the data/actions within the receipts themselves 😅

Reply:
I believe this particular function literally stores the `actions` of the `receipt`, so this function name is actually reflecting what it does. And the parent of all that actions is an `ActionReceipt`. 

Thus I think your suggestion is not going to work.